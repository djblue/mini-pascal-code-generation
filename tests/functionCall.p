program main;

class main 
begin
  var aa : integer;
  function go : integer;
  begin
    go := 64
  end;
  function main;
  begin
    aa := 1;
    print aa;
    aa := go();
    print aa;
    aa := aa + go() + 10;
    print aa
  end
end
.
