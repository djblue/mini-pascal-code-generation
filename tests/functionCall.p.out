1
64
138
